### DISCUSSION TITLE
get objects based on date using .NET SDK

### DISCUSSION THREAD

**Discussion by rgonzalez466** [minio/minio#17470] [Q&A] [answered]
I'm trying to create an application using Minio + Minio .NET SDK that'll retrieve all files uploaded in the past 5 minutes only, however, it seems like the only way is browse all the files and get their file timestamp which is quite inefficient or use mc console. Has anyone had any luck using a date parameter with their SDK ?

**Comment by harshavardhana** [comment, marked as answer]
There is no such S3 API, it is beyond the scope of this project.

**Reply by mimedo** [reply]
With CLI `mc` it's possible with `mc find --newer-than 5m`.
Do you have or are you planning to provide this same feature for min.io SDK?

**Reply by klauspost** [reply]
@mimedo mc lists all objects and does filtering on the result. You can do the same.
